src/modules/closePopUp.js

```diff
@@ -1,18 +1,10 @@
 const commentContainer = document.querySelector('.popUpComments');
-const mainDisplayContainer = document.querySelector('.mainDisplayContainer');
-const header = document.querySelector('.header');
-const footer = document.querySelector('footer');
 const closePopUp = () => {
   const crossBtn = document.querySelector('.cross');
   crossBtn.addEventListener('click', (e) => {
     e.preventDefault();
     commentContainer.style.display = 'none';
-    header.style.filter = 'blur(0)';
-    mainDisplayContainer.style.filter = 'blur(0)';
-    footer.style.filter = 'blur(0)';
   });
 };
 
-export {
-  closePopUp, commentContainer, mainDisplayContainer, header, footer,
-};
```

Comment:
It's good you changed the Comment counter file but the problem is it's my solo task. I don't know, Microverse can be asked us about it.